### DISCUSSION TITLE
Tell devs to run the CLI command for setting up tests

### DISCUSSION THREAD

**Discussion by dougstjohn** [craftcms/docs#302] [Ideas]
The 'Getting Started' page for testing is a lot of creating folders, creating files, and copying code.

It'd be simpler and less room for error if you tell devs to run the CLI command `./craft tests/setup` and then call out specific files they need to manually update. You could then remove the manual steps completely and just give a high-level overview of the folder structure.

Referencing: https://docs.craftcms.com/v3/testing/testing-craft/getting-started.html

**Comment by lindseydiloreto** [comment]
Wait, what?? `tests/setup` is a thing?
